DependencyInjection/Compiler/UnknownMiddleware.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace League\Tactician\Bundle\DependencyInjection\Compiler;
+
+use League\Tactician\Bundle\Middleware;
+
+class UnknownMiddleware extends \RuntimeException
```

Comment:
I have no control over merging the PR, I'm just trying to help out by reviewing so the swamped maintainers have less work. Given that the other exceptions in the bundle end with `Exception` (for example `InvalidCommandException`) I think it could help.